zio-http/shared/src/main/scala/zio/http/codec/HttpContentCodec.scala

```diff
@@ -193,7 +193,7 @@ object HttpContentCodec {
       ZPipeline.identity[Chunk[Byte]].flattenChunks
   }
 
-  val byteChunkCodec: HttpContentCodec[Chunk[Byte]] = {
+  implicit val byteChunkCodec: HttpContentCodec[Chunk[Byte]] = {
     HttpContentCodec(
       ListMap(
         MediaType.allMediaTypes.filter(_.binary).map(mt => mt -> ByteChunkBinaryCodec): _*,
```

Comment:
Given the order of the codecs matter, I wonder if `application/octet-stream` should come first here.

Reply:
I don't think so. I think it is like this. If you do not explicitly declare a media type, the accept header of the client should decide. That is the case already. If you want to return only `application/octet-stream`, then you have to make this explicit in the endpoint definition.